### PR TITLE
update public calendar policies

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

File: assets.md [hunk 1]
```diff
@@ -28,7 +28,14 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
 - [Calendar Google Group](https://groups.google.com/forum/#!forum/opentelemetry-calendar)
-    - Owner: Morgan McLean @mtwo
+    - Owner: @SergeyKanzhelev
+    - Write access controled via: opentelemetry-calendar-contributors@googlegroups.com
+
+- [opentelemetry-calendar-contributors Google Group](opentelemetry-calendar-contributors@googlegroups.com)
+    - Owners: current GC members.
+
+- [OpenTelemetry Calendar Invites Google Group](https://groups.google.com/g/opentelemetry-calendar)
+    - Owners: @mtwo
```
Comment (SergeyKanzhelev) [inline assets.md:38]: @mtwo perhaps you can make opentelemetry-calendar-contributors@googlegroups.com an admin for the Invites group? And make GC members as owners

Comment (mtwo) [reply assets.md:38]: Sure, doing it now

Comment (mtwo) [reply assets.md:38]: Looks like Google Groups doesn't like that:
```
Error! A group may not be the owner or manager of another group
```

File: assets.md [hunk 2]
```diff
@@ -28,7 +28,14 @@ This file is intended to list all the assets controlled by OpenTelemetry.
 ## Communication channels
 
 - [Calendar Google Group](https://groups.google.com/forum/#!forum/opentelemetry-calendar)
-    - Owner: Morgan McLean @mtwo
+    - Owner: @SergeyKanzhelev
+    - Write access controled via: opentelemetry-calendar-contributors@googlegroups.com
+
+- [opentelemetry-calendar-contributors Google Group](opentelemetry-calendar-contributors@googlegroups.com)
```
Comment (SergeyKanzhelev) [inline assets.md:34]:
```suggestion
- [opentelemetry-calendar-contributors Google Group](https://groups.google.com/g/opentelemetry-calendar-contributors)
```

File: assets.md [hunk 3]
```diff
@@ -28,7 +28,14 @@ This file is intended to list all the assets controlled by OpenTelemetry.
...
+    - Owners: current GC members.
+
```
Comment (SergeyKanzhelev) [inline assets.md:36]:
```suggestion
    - Owners: current GC members.
    - Members: SIG maintainers and individuals appointed by maintainers

```

File: assets.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply assets.md:39]: thinking about it I personally don't want to be a part of this group as I only import invites I need. Maybe you can keep yourself as a sole owner or maybe we need somebody from CNCF or from the current members as a backup

Comment (mtwo) [reply assets.md:39]: Do we have a CNCF admin email address that we can use for things like that? I think that we do (for the Twitter account, etc.) but I don't know the details (?). Alternatively I can just add all of the GC and / or TC members as admins?

Comment (SergeyKanzhelev) [reply assets.md:39]: @amye do you want to be a group owner?

@mtwo my problem owning this group is that it will flood my calendar with invites =).

**Review by alolita — APPROVED** [review]
+1

**Comment by SergeyKanzhelev** [comment]
Let's keep it this way for now. We can fix the ownership of invites google group later

**Pull request merged**